.gitignore

```diff
@@ -62,3 +62,9 @@ packages/coinstac-ui/coinstac-darwin*
 packages/coinstac-ui/coinstac-linux*
 packages/coinstac-ui/coinstac-win32*
 packages/coinstac-ui/coinstac-mas*
+
+# Package Lock Files
```

Comment:
as much as it sucks, we shouldn't be gitignoring lock files, as they are theoretically a repeatable template to npm i from